frontend/src/components/camp/picasso/DayResponsibles.vue

```diff
@@ -0,0 +1,123 @@
+<template>
+  <e-select
+    v-model="selectedCampCollaborations"
+    :items="availableCampCollaborations"
+    :loading="isSaving || isLoading ? 'secondary' : false"
+    :name="$tc('entity.day.fields.dayResponsibles')"
+    :error-messages="errorMessages"
+    outlined
+    :filled="false"
+    multiple
+    chips
+    deletable-chips
+    small-chips
+    v-bind="$attrs"
+    @input="onInput" />
+</template>
+
+<script>
+import serverErrorToString from '@/helpers/serverErrorToString.js'
+
+export default {
+  name: 'DayResponsibles',
+  props: {
+    // current period
+    period: {
+      type: Object,
+      required: true
+    },
+
+    // date of the DayEntity as ISO String
+    date: {
+      type: String,
+      required: true
+    }
+  },
+  data () {
+    return {
+      oldSelectedCampCollaborations: [],
+      selectedCampCollaborations: [],
+      errorMessages: [],
+      isSaving: false
+    }
+  },
+  computed: {
+    isLoading () {
+      return this.campCollaborations._meta.loading || this.period.days()._meta.loading || this.dayResponsibles._meta.loading
+    },
+    availableCampCollaborations () {
+      return this.campCollaborations.items.filter(cc => {
+        return (cc.status === 'established') || (this.currentCampCollaborationIRIs.includes(cc._meta.self))
```

Comment:
Auch hier: Warum lassen wir es nicht zu, eingeladene CampCollaborations als TC / TV einzutragen? Ich dachte wir hatten uns genau dafür entschieden dass das erlaubt sein sollte.
Ausgetretene Collaborations machen natürlich Sinn die hier auszuschliessen.

Reply:
Fixed in https://github.com/ecamp/ecamp3/pull/2404/commits/4657afd5ece3c5aa6a7f0dba2488e3cc50e9c71b